<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceInfoWidget</class>
 <widget class="QWidget" name="SourceInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>460</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>1</number>
     </property>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="HeaderLabel" name="label_2">
         <property name="text">
          <string>Recent Albums</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="GridView" name="recentAlbumView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>192</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="HeaderLabel" name="label">
           <property name="text">
            <string>Latest Additions</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TrackView" name="recentCollectionView"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="HeaderLabel" name="label_3">
           <property name="text">
            <string>Recently Played Tracks</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TrackView" name="historyView"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HeaderLabel</class>
   <extends>QLabel</extends>
   <header location="global">widgets/HeaderLabel.h</header>
  </customwidget>
  <customwidget>
   <class>GridView</class>
   <extends>QListView</extends>
   <header>playlist/GridView.h</header>
  </customwidget>
  <customwidget>
   <class>TrackView</class>
   <extends>QTreeView</extends>
   <header>playlist/TrackView.h</header>
  </customwidget>
  <customwidget>
   <class>TrackView</class>
   <extends>QTreeView</extends>
   <header>playlist/TrackView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
